Failed to install due to `postinstall` script on v0.7.3

Tring to install/upgrade to v0.7.3, `npm install` fails due to `postinstall` script added on 16dd116a53abee422b647022087f202c43b36708.

Does this necessary for users of this package?

```
$ npm i -D next-router-mock
npm ERR! code 2
npm ERR! path /home/mmyoji/tmp/node_modules/next-router-mock
npm ERR! command failed
npm ERR! command sh -c (cd test/next-10 && npm i); (cd test/next-11 && npm i); (cd test/next-12 && npm i); (cd test/next-canary && npm i)
npm ERR! sh: 1: cd: can't cd to test/next-10
npm ERR! sh: 1: cd: can't cd to test/next-11
npm ERR! sh: 1: cd: can't cd to test/next-12
npm ERR! sh: 1: cd: can't cd to test/next-canary

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mmyoji/.npm/_logs/2022-06-13T02_20_43_937Z-debug-0.log
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -21,7 +21,7 @@ jobs:
         command:
           - npm run test
           - npm run build
-          - npm run test:integration
+          - npm run test:integration:ci
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "build": "npm run clean && tsc",
     "clean": "rimraf dist",
     "build:watch": "tsc --watch",
-    "prepublishOnly": "npm run checks",
+    "prepublishOnly": "npm run build",
     "preversion": "npm run checks"
   },
   "repository": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "next-router-mock",
-  "version": "0.7.3",
+  "version": "0.7.4",
   "description": "Mock implementation of the Next.js Router",
   "main": "dist/index",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -4,11 +4,12 @@
   "description": "Mock implementation of the Next.js Router",
   "main": "dist/index",
   "scripts": {
-    "postinstall": "(cd test/next-10 && npm i); (cd test/next-11 && npm i); (cd test/next-12 && npm i); (cd test/next-canary && npm i)",
     "checks": "npm run docs && npm run test && npm run build && npm run test:integration",
     "docs": "doctoc README.md",
     "test": "jest",
     "test:watch": "jest --watch",
+    "test:integration:ci": "npm run test:integration:install && npm run test:integration",
+    "test:integration:install": "(cd test/next-10 && npm i); (cd test/next-11 && npm i); (cd test/next-12 && npm i); (cd test/next-canary && npm i)",
     "test:integration": "npm run build && npm run test:next-10 && npm run test:next-11 && npm run test:next-12 && npm run test:next-canary && npm run test:exports",
     "test:next-10": "cd test/next-10 && npm test",
     "test:next-11": "cd test/next-11 && npm test",

